tests/pyapi/test_job.py

```diff
@@ -157,3 +159,27 @@ def test_job_generic_resources(hq_env: HqEnv):
         "FINISHED",
         "WAITING",
     ]
+
+
+def test_task_priorities(hq_env: HqEnv):
+    (job, client) = prepare_job_client(hq_env)
+
+    priorities = list(range(-20, 20))
+    random.shuffle(priorities)
```

Comment:
Could you maybe just put a doc comment here to say what does the test do? It's not obvious to me. I would also rather just fix the seed to have it more deterministic.